keyboards/kbdfans/kbd6x/keymaps/via/rules.mk

```
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```

The keyboard config sets no action macro/function, and is causing a redefinition. 

It may be easiest to replace this line with: 
```suggestion
EXTRAFLAGS += -flto
TMK_COMMON_DEFS += -DLINK_TIME_OPTIMIZATION_ENABLE
```

---

I just tested it and it works. Great. suggestion, thank you!

---

Well, for reference, the LTO stuff is doing: 
https://github.com/qmk/qmk_firmware/blob/2fbbac04bb0c84323e3bb16b4d164d726046967c/tmk_core/common.mk#L158-L167

(And I'm the one that added it, too :) )